index.html

```diff
@@ -7,7 +7,7 @@
   <link rel="stylesheet" href="css/style.css">
   <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
   <title>Кекстаграм</title>
-  <script src="H:\рабочий стол\ProjectsWEB\2348239-kekstagram-3\js\maim.js"></script>
+  <script src="H:\ProjectsWEB\рабочий стол\2348239-kekstagram-3\js\main.js" type="module" ></script>
```

Comment:
Путь должен быть относительным
```suggestion
  <script src="\js\main.js" type="module" ></script>
```